src/LaunchDarklyApiClient.js

```diff
@@ -20,6 +22,13 @@ export class LaunchDarklyApiClient {
     static async create(API_TOKEN, log, swaggerYamlString) {
         log.debug(`creating api client with token: ${API_TOKEN}`);
 
+        let proxy = process.env.https_proxy;
```

Comment:
Possibly check for http_proxy/https_proxy as depending on the system either could be set?